<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassOkno</class>
 <widget class="QWidget" name="PassOkno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="login_dialog">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="vihod_knopka">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vvod_knopka">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>вход</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="parol_dialog">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="razg_dialog">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>66</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ЛОГИН:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>126</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ПАРОЛЬ:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
